Content.Shared/LieDown/SharedLieDownSystem.cs

```diff
@@ -0,0 +1,176 @@
+using Content.Shared.Actions;
+using Content.Shared.Examine;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Input;
+using Content.Shared.Interaction;
+using Content.Shared.Movement.Systems;
+using Content.Shared.Standing;
+using Content.Shared.Verbs;
+using Robust.Shared.Input.Binding;
+using Robust.Shared.IoC;
+using Robust.Shared.Player;
+
+namespace Content.Shared.LieDown
+{
+    public class SharedLieDownSystem : EntitySystem
+    {
+        [Dependency] private readonly SharedActionsSystem _actions = default!;
+        [Dependency] private readonly MovementSpeedModifierSystem _movement = default!;
+        [Dependency] private readonly StandingStateSystem _standing = default!;
+
+        public override void Initialize()
+        {
+            SubscribeLocalEvent<LyingDownComponent, InteractHandEvent>(OnInteractHand);
+            SubscribeLocalEvent<LyingDownComponent, GetVerbsEvent<AlternativeVerb>>(AddStandUpVerb);
+            SubscribeLocalEvent<LyingDownComponent, ExaminedEvent>(OnExamined);
+            SubscribeLocalEvent<LyingDownComponent, RefreshMovementSpeedModifiersEvent>(OnRefresh);
+
+            SubscribeLocalEvent<LyingDownComponent, ComponentStartup>(OnComponentStartup);
+            SubscribeLocalEvent<LyingDownComponent, ComponentShutdown>(OnComponentShutdown);
+
+            // Bind keybinds to lie down action
+            SubscribeNetworkEvent<ChangeStandingStateEvent>(OnChangeAction);
+            CommandBinds.Builder
+                .Bind(ContentKeyFunctions.LieDownStandUp, InputCmdHandler.FromDelegate(ChangeLyingState))
+                .Register<SharedLieDownSystem>();
+        }
+
+        private void OnComponentShutdown(EntityUid uid, LyingDownComponent component, ComponentShutdown args)
```

Comment:
```suggestion
        private void OnComponentShutdown(Entity<LyingDownComponent> lying, ref ComponentShutdown args)
```

Сска уже полгода как перешла на новый формат подписок на ивенты. Старый рано или поздно устареет, и от него стоит избавляться уже сейчас

Reply:
uid получается напрямую из lying
component из lying.Comp

Reply:
Это касается вообще всех подписок во всех системах